### PR TITLE
Third party dependency update

### PR DESCRIPTION
**Pull request by rogersteblerbsi**

### CODE REVIEW

File: org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java [hunk 1]
```diff
@@ -318,7 +318,6 @@ protected Path resolveKeyStorePath(String path) {
    */
   protected void installErrorHandler(Server server) {
     ErrorHandler handler = new ErrorHandler();
-    handler.setShowServlet(false);
```
Comment (matthiaso) [inline org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java:321]: For some classes we seem to use a ee10 variant, for this class we do not use `org.eclipse.jetty.ee10.servlet.ErrorHandler`

Is there a pattern/when do we use the ee10 variant?

File: org.eclipse.scout.rt/pom.xml [hunk 2]
```diff
@@ -123,19 +123,19 @@
     <scout.base.version>25.1.0</scout.base.version>
     <base.version>${scout.base.version}</base.version>
     <org.eclipse.scout.rt.version>${project.version}</org.eclipse.scout.rt.version>
-    <jetty.version>11.0.24</jetty.version>
+    <jetty.version>12.0.15</jetty.version>
```
Comment (matthiaso) [inline org.eclipse.scout.rt/pom.xml:126]: 12.0.16 has been released last week, do we want to upgrade to the latest and greatest?

File: org.eclipse.scout.rt/pom.xml [hunk 3]
```diff
@@ -123,19 +123,19 @@
...
     <slf4j.version>2.0.9</slf4j.version>
     <logback.version>1.4.14</logback.version>
-    <jackson.version>2.16.0</jackson.version>
-    <io.netty-version>4.1.114.Final</io.netty-version>
+    <jackson.version>2.18.2</jackson.version>
+    <io.netty-version>4.1.115.Final</io.netty-version>
```
Comment (matthiaso) [inline org.eclipse.scout.rt/pom.xml:130]: 4.1.116.Final seems to be available, do we want to use latest and greatest?

File: org.eclipse.scout.rt/pom.xml [hunk 4]
```diff
@@ -123,19 +123,19 @@
...
     <apache.tika-version>3.0.0</apache.tika-version>
     <batik.version>1.18</batik.version>
     <activemq.version>6.1.4</activemq.version>
     <artemis.version>2.38.0</artemis.version>
-    <com.google.http-client.version>1.43.3</com.google.http-client.version>
-    <org.apache.httpcomponents.version>5.2.1</org.apache.httpcomponents.version>
-    <opentelemetry.version>1.34.1</opentelemetry.version>
-    <opentelemetry.instrumentation.version>2.0.0-alpha</opentelemetry.instrumentation.version>
+    <com.google.http-client.version>1.45.2</com.google.http-client.version>
```
Comment (matthiaso) [inline org.eclipse.scout.rt/pom.xml:135]: 1.45.3?

File: pom.xml [hunk 5]
```diff
@@ -14,7 +14,7 @@
   <parent>
     <groupId>org.eclipse.scout</groupId>
     <artifactId>maven_rt_plugin_config-master</artifactId>
-    <version>25.1-SNAPSHOT</version>
+    <version>25.1-rsb-SNAPSHOT</version>
```
Comment (matthiaso) [inline pom.xml]: Remove this (everywhere)

**Review by matthiaso — APPROVED** [review]

File: org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java (hunk 1 above)
Comment (rogersteblerbsi) [reply org.eclipse.scout.rt.app/src/main/java/org/eclipse/scout/rt/app/Application.java:321]: Also discussed with Paolo. For now we keep the one from the server module (org.eclipse.jetty.server.handler.ErrorHandler)

**Pull request merged**